Samples/src/FSharpWithQSharp/README.md

```diff
@@ -0,0 +1,41 @@
+# Using Q# with F#
+
+This sample shows how to use Q# code with F# classical driver. 
+
+Q# code is compiled to C# for the purposes of local simulation, so using Q# with C# is very straightforward, as described in the documentation. Using Q# with F# is a little less obvious, since you can't mix Q# and F# files in the same project. You have to create a Q# library (which will yield a .csproj project with only Q# files in it) and to reference it from a purely F# application.
```

Comment:
```suggestion
Q# code is compiled to C# for the purposes of local simulation, so using Q# with C# is very straightforward, as described in the documentation. Using Q# with F# is a little less obvious, since you can't mix Q# and F# files in the same project. You have to create a Q# library (which will yield a .csproj project with only Q# files in it) and then reference it from a purely F# application.
```